rm102/gui.py

```diff
@@ -83,13 +103,14 @@ def loadButtonClicked(self):
             return
         with open(fname[0], 'r') as fobj:
             i = 0
-            for line in fobj.readlines():
+            lines = fobj.readlines()
+            for line in lines:
                 self.commandRegisterList.model.item(i).setText(line.strip('\n'))
                 i += 1
-                if self.commandRegisterList.model.item(i) is None:
+                if self.commandRegisterList.model.item(i) is None and i < len(lines):
```

Comment:
Imho an empty line at the end of the CommandRegisterList a good and wanted thing.

Reply:
Reverted in a02275279ec5adca4306b11867d0e5a32ebb5038